DSpace/dspace-angular
# Media viewer thumbnails don't take viewport width into account

## References
* Fixes #1904

## Description
Fixed the issue which caused both the media viewer thumbnail for images and files, and the video player to overflow outside of their column on item pages.

## Instructions for Reviewers

List of changes in this PR:
- `MediaViewerImageComponent`:
  * Removed fixed width
  * Gave it the width of the column by default
  * Ensured empty space is always generated underneath the image (is necessary because of the way ngx-gallery displays the images (they use background-image instead of `<img>` tags))
- `MediaViewerVideoComponent`: 
  * Removed fixed width
  * Gave it the width of the column by default

**Guidance for how to test or review this PR:**
- Enable the mediaViewer in your config by adding this to `config.yml`:
  ```yml
  mediaViewer:
    image: true
    video: true
  ```
- Create a new item with only an image in the `ORIGINAL` bundle.
- Create a new item with only a video in the `ORIGINAL` bundle.
- Create a new item with only a file (random non-media file like an xml file) in the `ORIGINAL` bundle.

➜ When your screen is between 768px and 1200px check for each of the items that the image or video player is not displayed like this:
<img src="https://user-images.githubusercontent.com/31547038/196149786-14d81203-a5ea-4721-9d9f-dfe154c8c57f.png" width="50%">

## Checklist
